.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
+  "recommendations": [
+    "editorconfig.editorconfig",
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode"
+  ]
```

Comment:
Can you add "Code Spell Checker" too? `"streetsidesoftware.code-spell-checker"` to help people catch spelling mistakes in comments, variable names, etc.  It's great!

Reply:
Definitely!